packages/stryker/test/unit/SandboxPoolSpec.ts

```diff
@@ -34,15 +34,15 @@ describe('SandboxPool', () => {
       .onCall(1).resolves(secondSandbox);
 
     expectedInputFiles = [file()];
-    sut = new SandboxPool(options, expectedTestFramework, expectedInputFiles);
+    sut = new SandboxPool(options, expectedTestFramework, expectedInputFiles, 42);
```

Comment:
I get that 42 is the answer to everything, but could you make it a constant, especially because it's used in multiple places?